taskt/Core/Automation/Commands/Data Commands/FormatDataCommand.cs

```diff
@@ -97,32 +95,26 @@ public override void RunCommand(object sender)
             }
             else
             {
-                formattedString.StoreInUserVariable(sender, v_applyToVariableName);
+                formattedString.StoreInUserVariable(sender, v_OutputUserVariableName);
             }
-
-
-
         }
+
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
             //create standard group controls
-            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InputValue", this, editor));
-            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_FormatType", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InputData", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_FormatType", this, editor));
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_ToStringFormat", this, editor));
-
-            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_applyToVariableName", this));
-            var VariableNameControl = CommandControls.CreateStandardComboboxFor("v_applyToVariableName", this).AddVariableNames(editor);
-            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_applyToVariableName", this, new Control[] { VariableNameControl }, editor));
-            RenderedControls.Add(VariableNameControl);
+            RenderedControls.AddRange(CommandControls.CreateDefaultOutputGroupFor("v_OutputUserVariableName", this, editor));
 
             return RenderedControls;
-
         }
+
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [Format '" + v_InputValue + "' and Apply Result to Variable '" + v_applyToVariableName + "']";
+            return base.GetDisplayValue() + $" [Format '{v_InputData}' - Apply Result to Variable '{v_OutputUserVariableName}']";
```

Comment:
"- Store Data in..."

Reply:
Also this needs to be more specific I think. $" [Format {v_FormatType} '{v_InputData}' As '{v_ToStringFormat}' -  Store {v_InputData} in '{v_OutputUserVariableName}']"

Reply:
done.
Display value is slightly changed like:
$" [Format '{v_InputData}' as '{v_ToStringFormat}' - Store Formatted {v_FormatType} in '{v_OutputDataVariableName}']"